packages/core/src/redocly/registry-api.ts

```diff
@@ -23,7 +25,10 @@ export class RegistryApi {
   }
 
   private async request(path = '', options: RequestInit = {}, region?: Region) {
-    const headers = Object.assign({}, options.headers || {}, { 'x-redocly-cli-version': version });
+    const headers = Object.assign({}, options.headers || {}, {
+      'x-redocly-cli-version': version,
```

Comment:
Should we also put the same [here](https://github.com/Redocly/redocly-cli/blob/master/packages/cli/src/commands/push.ts#L371)?

Reply:
We may but it's not essential. This goes directly to s3.

Reply:
Got it